client/src/components/Donation/components/DonateFormChildViewForHOC.js

```diff
@@ -130,6 +131,11 @@ class DonateFormChildViewForHOC extends Component {
     // scroll to top
     window.scrollTo(0, 0);
 
+    ga.event({
+      category: 'Donation',
+      action: 'Stripe Form Submission'
+    });
```

Comment:
This will make it common amongst all the types of submissions. Also you should add events for visits to the modal, the donate page and the certification page. Without it there is nothing to compare the data against?

Reply:
Also, I am not sure that view itself is a good place have this. It should probably be somewhere else?

Reply:
Good points, I will specify the GA's and have them in their parent component or page.

Also, would you like to have them fire during processing or success?

Do we need GA's for error?

Reply:
I do not think the logic should be in any views, should it? I would prefer them to be agnostic of the views.

Reply:
we could have them fire from a saga? I will look around for best practices.